dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -491,6 +580,22 @@ void resolveFeaturesAsSet() {
         }
     }
 
+    /**
+     * Record Versionless Messages and Issues
+     *
+     * @param result
+     * @param resolverRepo
+     * @param requestedFeatures
+     */
+    private void recordVersionless(Result result, Repository resolverRepo) {
+
+        resolvedPlatforms = result.getResolvedPlatforms();
+        missingPlatforms = result.getMissingPlatforms();
+        if (resolvedPlatforms.isEmpty() || !missingPlatforms.isEmpty())
+            hasVersionlessIssue = true;
+
```

Comment:
For consistency with the rest of the code, please use braces even when the if-block only contains one statement.